src/main/java/smartthings/konsumer/KafkaListener.java

```diff
@@ -18,32 +21,38 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import java.util.EventListener;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
 import java.util.concurrent.*;
 
-public class KafkaListener {
+public class KafkaListener<K, V, R> {
```

Comment:
Might be good to document what the generics are here are so you can reference quickly?